packages/calcite-components/src/utils/floating-ui.ts

```diff
@@ -406,6 +413,18 @@ export function getEffectivePlacement(floatingEl: HTMLElement, placement: Logica
   return placement.replace(/leading/gi, placements[0]).replace(/trailing/gi, placements[1]) as EffectivePlacement;
 }
 
+const hiddenFloatingElStyle = {
+  visibility: "hidden",
+  pointerEvents: "none",
+  // initial positioning based on https://floating-ui.com/docs/computePosition#initial-layout
+  position: "absolute",
+  transform: "",
+  top: "",
+  left: "",
+  insetInlineStart: "-999999px",
+  insetBlockStart: "-999999px",
```

Comment:
using this will make sure its positioned off the screen no matter what page direction or viewport size. Therefore, it shouldn't cause any scrollbars or width change in any parent element.